### PR TITLE
TC_15.01.002 | Footer> Access to REST API > Verify redirection to REST API page

### PR DESCRIPTION
**Pull request by gulkinna**
https://trello.com/c/Gs7xBYAG

### CODE REVIEW

**Comment by AnnieZ1207** [comment]
would be nice to add check that redirected page url contains "/api" as it said at US https://trello.com/c/18GEystq/48-us1501-footer-access-to-rest-api

File: cypress/e2e/redirectionToRESTAPIPage.cy.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/// <reference types="cypress"/>
+
+describe('redirectionToRESTAPIPage', () => {
+    it('TC_15.01.002 | Footer> Access to REST API > Verify redirection to REST API page', () => {
+        cy.get('.rest-api').click()
+
+        cy.get('#page-body h1').should('be.visible').and('have.text', 'REST API')
```
Comment (S-Lanchik) [inline cypress/e2e/redirectionToRESTAPIPage.cy.js:7]: Put 'REST API' to fixtures.

File: cypress/e2e/redirectionToRESTAPIPage.cy.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        cy.get('.jenkins-breadcrumbs__list-item a[href="/api/"]').should('be.visible').and('have.text', 'API')
```
Comment (S-Lanchik) [inline cypress/e2e/redirectionToRESTAPIPage.cy.js:8]: Put 'API' to fixtures.

File: cypress/e2e/redirectionToRESTAPIPage.cy.js (hunk 1 above)
Comment (gulkinna) [reply cypress/e2e/redirectionToRESTAPIPage.cy.js]: Done

**Review by S-Lanchik — APPROVED** [review]

**Review by LenaDik — APPROVED** [review]

**Pull request merged**